jenkins-integration/jenkins-jobs/ops-deployment/run_ops_deployment.yaml

```diff
@@ -12,24 +12,26 @@
     - string:
         name: PUPPET_GATLING_MASTER_BASE_URL
         description: "The URL of the puppet master SUT"
+        default: "puppetserver-perf-sut54.delivery.puppetlabs.net"
 
     scm:
       - git:
-          url: https://github.com/puppetlabs/gatling-puppet-load-test.git
+          url: https://github.com/cprice404/gatling-puppet-load-test.git
```

Comment:
I did, for now.  Though that is no longer the branch I want, I think, so thanks :)